modules/netstat/netstat_linux.go

```diff
@@ -9,15 +9,152 @@ package netstat /* import "mig.ninja/mig/modules/netstat" */
 import (
 	"bufio"
 	"fmt"
+	"io/ioutil"
 	"net"
 	"os"
+	"path"
 	"regexp"
 	"strconv"
 	"strings"
 )
 
-// HasSeenMac on linux looks for a matching mac address in /proc/net/arp
-// and returns its MAC and IP address if found
+// Represents lines obtained from the /proc file system related to network
+// activity
+type procNetLine struct {
+	line         string // The line from the proc file
+	nsIdentifier string // A namespace identifier associated with the process
+}
+
+// Return process candidates organized by active network namespace, this will
+// include the host namespace in addition to other namespaces
+func procCandidates() (map[string][]string, error) {
+	var err error
+	ret := make(map[string][]string)
```

Comment:
so, if I understand correctly, this is a map of ns ids, where each ns has a list of processes. correct?
```json
{
  "4026531957": [ "/proc/1/net", "/proc/123/net" ],
   "8716321": [ "/proc/42/net", "/proc/51/net" ]
}
```